src/js/components/inspectors/Text.jsx

```diff
@@ -0,0 +1,133 @@
+'use strict';
+var React = require('react'),
+    Property = require('./Property'),
+    ExtentProperty = require('./ExtentProperty');
+
+var Text = React.createClass({
+  render: function() {
+    var props = this.props,
+        primitive = props.primitive,
+        update = primitive.properties.update;
+
+    return (
+      <div>
+        <h3>Text</h3>
+
+        <Property name="text" label="Text"
+          type="text"
+          primitive={primitive}
+          canDrop={true}
+          field={update.text.field}
+          scale={update.text.scale}
+          signal={update.text.signal} />
+
+        <h3>Font</h3>
+
+        <Property name="font" label="Style"
+          primitive={primitive}
+          type="select"
+          opts={['Helvetica', 'Verdana', 'Georgia', 'Palatino', 'Garamond', 'Trebuchet MS']}
```

Comment:
I don't know if this matters, but symbol mark contains all it's special info where the mark is defined. 
https://github.com/vega/lyra/blob/lyra2/src/js/model/primitives/marks/Symbol.js#L56

called in the inspector here: https://github.com/vega/lyra/blob/lyra2/src/js/components/inspectors/Symbol.jsx#L35